pennylane/qchem/factorization.py

```diff
@@ -56,7 +124,7 @@ def factorize(two_electron, tol_factor=1.0e-5, tol_eigval=1.0e-5, cholesky=False
     >>> mol = qml.qchem.Molecule(symbols, geometry)
     >>> core, one, two = qml.qchem.electron_integrals(mol)()
     >>> two = np.swapaxes(two, 1, 3) # convert to chemist notation
-    >>> factors, eigvals, eigvecs = factorize(two, 1e-5, 1e-5)
+    >>> factors, cores, leaves = factorize(two, 1e-5, 1e-5)
```

Comment:
I get slightly different values for the example, please double check and update if needed.